airflow/ui/src/components/TriggerDag/TriggerDAGForm.tsx

```diff
@@ -139,7 +129,7 @@ const TriggerDAGForm = ({ dagId, onClose, open }: TriggerDAGFormProps) => {
         size="lg"
         variant="enclosed"
       >
-        <FlexibleForm params={paramsDict} />
+        <FlexibleForm initialParamsDict={initialParamsDict} />
```

Comment:
Since we have this now. I think we should remove the ConfigurationJson under Advanced

Reply:
I think we should keep the advanced JSON configuration. Because it is an alternative to paste a prepared config (which you can copy from a previous DAG run) w/o the need to fill all fields manually. Saves ~20+ clicks for power users.